app/views/activities/_activity_feed.html.erb

```diff
@@ -18,14 +18,10 @@
       </div>
       <ul class="activity-feed__list js-poller"
         data-interval="<%= poll_interval_milliseconds %>"
-        data-url="<%= booking_request_activities_path(booking_request) %>">
-        <%= render activities.first if activities.present? %>
-      </ul>
+        data-url="<%= booking_request_activities_path(booking_request) %>"><%= render activities.first if activities.present? %></ul>
```

Comment:
Had to remove the HTML line break formatting here to ensure that the :empty css attribute is triggered (which stops two horizontal lines from appearing)